generate_test_data.py

```diff
@@ -0,0 +1,32 @@
+from git import Repo
+import shutil
+from clinical_etl import CSVConvert
+import argparse
+import os
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description="A script that ingests clinical data into Katsu")
+    parser.add_argument("--output", help="Path to clone synth data repo.", required=True)
+    return parser.parse_args()
+
+
+def main(args):
+    ingest_repo_dir = os.path.dirname(os.path.abspath(__file__))
+    print(f"Cloning mohccn-synthetic-data repo into {args.output}")
+    Repo.clone_from("https://github.com/CanDIG/mohccn-synthetic-data.git", args.output)
```

Comment:
does Repo do some sort of magic re: authentication when cloning?

Reply:
I don't think so, because it is a public repo there isn't any auth required. opted for a python method instead subprocess so that the csv conversion doesn't start until after the repo is cloned.